plugins/github-pull-requests-board/src/utils/types.tsx

```diff
@@ -83,6 +112,8 @@ export type PullRequestsColumn = {
   content: PullRequests;
 };
 
-export type PRCardFormating = 'compacted' | 'fullscreen' | 'draft';
+export type PRCardFormating = 'compacted' | 'fullscreen' | 'draft' | 'team';
```

Comment:
I find it odd that this mixes "shape of the display" with "what to display".

Reply:
👀 Yeah, kinda agree

I've looked into it, and it seems the `compacted` option is never used anywhere. This leaves `fullscreen`, `draft`, and the new `team`. These are part of what we could call an "options" menu, but it's currently labeled as `infoCardFormat` and `PRCardFormating` in the code base.

I'm thinking that we can do the following:
* Drop the `compacted` option since it's unused anywhere
* Rename things to make it clear that these are options, not "formatting" (or at least, not only formatting):
  * `PRCardFormating` could become `PRCardOptions`
  * `infoCardFormat` could become `infoCardOptions` 

Alternatively, I can completely break down PRCardFormatting to separate "formating" from "filters". We'd have:
* `PRCardFormatting`, with only `['fullscreen']` (since `compacted` is unused)
* `PRCardFilters` with `['draft', 'team']`
* Might make things awkward since we currently have `const [infoCardFormat, setInfoCardFormat] = useState<PRCardFormating[]>([]);` though 🤔 

What do you think?